Sources/AudioPlayer.swift

```diff
@@ -24,8 +26,14 @@ public final class AudioPlayer: ObservableObject {
         set { synchronizer?.isMuted = newValue }
     }
 
-    public init(timeUpdateInterval: CMTime = CMTime(value: 1, timescale: 10)) {
+    public init(
+      timeUpdateInterval: CMTime = CMTime(value: 1, timescale: 10),
+      didStartPlaying: @escaping @Sendable () -> Void,
+      didFinishPlaying: @escaping @Sendable () -> Void
```

Comment:
Can we add default parameters to these closures so they can be omitted when not needed?

```suggestion
      didStartPlaying: @escaping @Sendable () -> Void = {},
      didFinishPlaying: @escaping @Sendable () -> Void = {}
```